src/cljs/athens/views/blocks/editor.cljs

```diff
@@ -173,6 +178,18 @@
                                    (if (true? linked-ref)
                                      (rf/dispatch [::linked-ref.events/toggle-open! uid])
                                      (toggle uid (not open))))}])
+
+          (when key
+            [:> PropertyName {:name    (:node/title key)
+                              :onClick (fn [e]
+                                         (let [shift? (.-shiftKey e)]
+                                           (rf/dispatch [:reporting/navigation {:source :block-property
```

Comment:
self-nit: nav reporting at this level fells wrong.
addition args to `router/navigate-*` is also imperfect, but maybe perfectly imperfect?

Reply:
I didn't quite like this either. Felt like a chore that I had to figure out and get right.